tests/queries/0_stateless/03010_view_prewhere_in.sql

```diff
@@ -0,0 +1,17 @@
+DROP TABLE IF EXISTS v;
+CREATE VIEW v (`date` UInt32,`value` UInt8) AS
+WITH
+    data AS (SELECT '' id LIMIT 0),
+    r AS (SELECT'' as id, 1::UInt8 as value)
+SELECT
+    now() as date,
```

Comment:
```suggestion
    1::UInt32 as date,
```

To use correct types. In the current release it reproduces:
```
Code: 352. DB::Exception: Received from clickhouse-01:49000. DB::Exception: Block structure mismatch in (columns with identical name must have identical structure) stream: different columns:
date UInt32 UInt32(size = 0)
date UInt32 Const(size = 0, UInt32(size = 1)). (AMBIGUOUS_COLUMN_NAME)
```